src/components/Map/Map.tsx

```diff
@@ -15,7 +15,7 @@ export interface MapProps extends Omit<kakao.maps.MapOptions, 'center' | 'level'
   /**
    * 지도 크기
    */
-  size: number | { width: number; height: number };
+  size: number | { width: number | `${number}%`; height: number | `${number}%` };
```

Comment:
`style`에 넣고 있어서 아래와 같이 해도 좋을 거 같은데 괜찮으시다면 반영 부탁드림다 🙏 

```suggestion
  size: number | { width: CSSProperties['width']; height: CSSProperties['height'] };
```

Reply:
[1b11dc0](https://github.com/project-kkini/kkini-front/pull/12/commits/1b11dc0f65afc451dbd7dd8f593c3374807f7475) 요기서 반영했습니다!